add js library (#30)
huggingface/hf-endpoints-documentation: docs/source/guides/test_endpoint.mdx

# Send Requests to Endpoints

You can send requests to Inference Endpoints using the UI leveraging the Inference Widget or programmatically, e.g. with cURL or the `@huggingface/inference` library. Below you can find examples for each of these methods.

> ## Use the UI to send request

Reviewer: ```suggestion
## Use the UI to send requests
```

```

To use your cURL command as code, use the [cURL Converter](https://curlconverter.com/) tool to quickly get started with the programming language of your choice.

> ## Use javascript library `@huggingface/inference`

Reviewer: ```suggestion
## Use `@huggingface/inference`
```

Reviewer: or maybe "Use the javascript library"

> here's how you can call an inference endpoint:

Reviewer: ```suggestion
You can use the javascript library to call an inference endpoint:
```

```ts
const inference = new HfInference('hf_...') // your user token

const gpt2 = inference.endpoint('https://xyz.eu-west-1.aws.endpoints.huggingface.cloud/gpt2-endpoint')
const { generated_text } = await gpt2.textGeneration({ inputs: 'The answer to the universe is' })
```

> **custom handler**
> 
> `@huggingface/inference` supports tasks from https://huggingface.co/tasks, and is typed accordingly. But what if your model as additional inputs, or even custom inputs or outputs?
> 
> You can use `.request` and `.streamingRequest` now!

Reviewer 2: ```suggestion
### Custom handler

`@huggingface/inference` supports tasks from https://huggingface.co/tasks, and is typed accordingly. 

If your model has additional inputs, or even custom inputs / outputs you can use the more generic `.request` / `streamingRequest`:
```

The release notes are in a "peppy" tone, the doc should be more relaxed / neutral :)
